cvat/apps/engine/management/commands/export_completed_annotations.py

```diff
@@ -74,3 +74,12 @@ def dump_annotations(task, dump_folder, overwrite=False):
     if not os.path.exists(output_folder):
         os.makedirs(output_folder)
     dump_annotation_for_task(task, output_folder, overwrite=overwrite)
+
+def delete_task(task):
+    delete_task_data(task.id, task.assignee)
+
+    shutil.rmtree(task.get_task_dirname(), ignore_errors=True)
+    shutil.rmtree(task.data.get_data_dirname(), ignore_errors=True)
+
+    task.data.delete()
+    task.delete()
```

Comment:
whats the difference between task.data.delete() and delete_task_data(task.id, task.assignee)?

what about something like

```
def delete_task(task):
    try:
        delete_task_data(task.id, task.assignee)
        shutil.rmtree(task.get_task_dirname(), ignore_errors=True)
        shutil.rmtree(task.data.get_data_dirname(), ignore_errors=True)
        task.data.delete()
    except Exception as e: 
        print(f"Could not delete task data for {task}!")
        print(e)
    else:
        task.delete()
```

Reply:
- `delete_task_data` is meant for removing annotations (and annotations data) related to this task, it is implemented on `cvat/apps/engine/annotation.py`.
- `task.data.delete()` will only remove this data entry from the db.

Reply:
ah ok thanks. 
the exception handling is unnecessary I guess..
I think I can merge right?